Formula/doctl.rb

```diff
@@ -23,11 +23,18 @@ def install
     ln_sf buildpath, buildpath/"src/github.com/digitalocean/doctl"
     Language::Go.stage_deps resources, buildpath/"src"
 
-    system "go", "build", "-ldflags", "-X github.com/digitalocean/doctl/Build=#{version}", "github.com/digitalocean/doctl/cmd/doctl"
+    doctl_version = version.to_s.split(/\./)
+    base_flag = "-X github.com/digitalocean/doctl"
+    ldflags = "#{base_flag}.Major=#{doctl_version[0]}"
```

Comment:
The `ldflags` var is all one string, not individual args. If you do `*args`, it'll create them as separate items